src/Services/Processor/Substrate/Events/Implementations/Marketplace/ListingCreated.php

```diff
@@ -7,7 +7,7 @@
 use Enjin\Platform\Marketplace\Enums\ListingState;
 use Enjin\Platform\Marketplace\Enums\ListingType;
 use Enjin\Platform\Marketplace\Events\Substrate\Marketplace\ListingCreated as ListingCreatedEvent;
-use Enjin\Platform\Marketplace\Models\MarketplaceListing;
+use Enjin\Platform\Marketplace\Models\Laravel\MarketplaceListing;
```

Comment:
```suggestion
use Enjin\Platform\Marketplace\Models\MarketplaceListing;
```
Can we use the Model Resolver version here?

Reply:
Doesn't work, will give a type error. Would have to use just Model but that takes type hint away

Reply:
That's fine, we use Model elsewhere for this reason too so should try to keep it consistent 👍🏻